mods/tuxemon/db/technique/flamethrower.json

```diff
@@ -1,7 +1,7 @@
 {
   "tech_id": 73,
   "accuracy": 0.8,
-  "animation": "fire lion right",
+  "animation": "firelion_right",
```

Comment:
From what I saw, there is more slip-ups like this (spaces in slugs, wrong names, etc.). Should I clean these up in this PR or are we going to tackle them later once we have a rundown on the missing credits/attributions and non-64x64 anims?

Reply:
Fix as many as you can

Reply:
Done!